File: Julia/functions/genParams.jl
function genParams()

    # define parameters
    n_sim = 10^4
    n_items = 6
    
    alpha = 10.0
    gamma = 0.9
    sigma = 0.5
    
    delta = 0.8
    
    # store them into structs
    S = Sng(n_sim, n_items)
    F = Free(alpha, gamma, sigma, delta)

    return S::Sng, F::Free

end

File: MATLAB/functions/genParams.m
function [S, F] = genParams()

    % create struct for fixed parameters
    S.n_sim = 10^4;
    S.n_items = 6;

    % free parameters
    F.alpha = 10.0;
    F.gamma = 0.9;
    F.sigma = 0.5;
    F.delta = 0.8;

end